src/main/java/com/rultor/agents/github/qtn/QnReferredTo.java

```diff
@@ -54,6 +56,16 @@
 public final class QnReferredTo implements Question {
 
     /**
+     * Regex that matches the @ and Rultor user login in Github comments.
+     * Matches the login when at the beginning of the comment string or when
+     * preceded by a space or comma.
```

Comment:
@original-brownbear I'm not sure whether we need to detect e.g. `,@rultor`, but it doesn't hurt either